govc/test/namespace.bats

```diff
@@ -113,4 +113,140 @@ load test_helper
     run govc namespace.service.info -json service2
     assert_matches DE-ACTIVATED
 
+}
+
+@test "namespace.create" {
+    vcsim_env
+
+    run govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-1
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-1 | jq)
+    assert_equal "domain-c1" $(echo $ns | jq -r '."cluster"')
+    assert_equal "0" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+
+    run govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-2 -content-libraries=lib1,lib2
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-2 | jq)
+    assert_equal "2" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+}
+
+@test "namespace.change" {
+    vcsim_env
+
+    govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-1
+
+    run govc namespace.change -content-libraries=lib1,lib2 -vm-classes=class1 test-namespace-1
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-1 | jq)
+    assert_equal "2" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+    assert_equal "lib1" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"[0]')
+    assert_equal "lib2" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"[1]')
+    assert_equal "1" $(echo $ns | jq -r '."vm_service_spec"."vm_classes"' | jq length)
+    assert_equal "class1" $(echo $ns | jq -r '."vm_service_spec"."vm_classes"[0]')
+}
+
+@test "namespace.info" {
+    vcsim_env
+
+    govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-1
+
+    ns=$(govc namespace.info test-namespace-1 | jq)
+    assert_equal "domain-c1" $(echo $ns | jq -r '."cluster"')
+    run govc namespace.info non-existing-namespace
+    assert_failure
+}
+
+@test "namespace.ls" {
+    vcsim_env
+
+    ls=$(govc namespace.ls)
+    assert_equal "0" $(echo $ls | jq length)
+
+    govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-1
+    ls=$(govc namespace.ls)
+    assert_equal "1" $(echo $ls | jq length)
```

Comment:
can we check the namespace name is correct too?

Reply:
Sure, I've added a validation for that